control/veda_file_discovery.py

```diff
@@ -125,162 +127,199 @@ def about_video_validate(self, meta, key):
 
         reset_queries()
 
-    def studio_s3_ingest(self):
+    def move(self, key, destination_dir):
```

Comment:
a more specific name would be better? `move_video` or something like that

Reply:
okay

Reply:
This change hasn't been pushed ;)